lib/server.js

```diff
@@ -891,13 +891,15 @@ Server.prototype._run = function _run(req, res, route, chain, cb) {
                 return (next());
             }
 
+            handlerName = (chain[i].handlerName ||
```

Comment:
That's right, this handlerName property should be mentioned in the docs.
I'm not really sure where and how to add this in the doc, and as I'm not native english speaker I'm not sure I'll do it right.... but I'll try something ;)